model/daal4py/examples/lr/train_data.sh

```diff
@@ -0,0 +1,19 @@
+cat > train.csv << EOF
+f1,ans
+12.4, 11.2
```

Comment:
I think we need to remove spaces here, likely the cause of:

```

2020-07-18T03:21:53.6184466Z ======================================================================
2020-07-18T03:21:53.6184983Z ERROR: test_00_train (tests.test_lr.TestDAAL4PyLRModel)
2020-07-18T03:21:53.6185693Z ----------------------------------------------------------------------
2020-07-18T03:21:53.6186200Z Traceback (most recent call last):
2020-07-18T03:21:53.6187607Z   File "/home/runner/work/dffml/dffml/dffml/util/asynctestcase.py", line 69, in run_it
2020-07-18T03:21:53.6187989Z     result = self.loop.run_until_complete(coro(*args, **kwargs))
2020-07-18T03:21:53.6188385Z   File "/usr/share/miniconda/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
2020-07-18T03:21:53.6188676Z     return future.result()
2020-07-18T03:21:53.6188995Z   File "/home/runner/work/dffml/dffml/model/daal4py/tests/test_lr.py", line 64, in test_00_train
2020-07-18T03:21:53.6189288Z     await train(self.model, *self.train_data)
2020-07-18T03:21:53.6189591Z   File "/home/runner/work/dffml/dffml/dffml/high_level.py", line 380, in train
2020-07-18T03:21:53.6189873Z     return await mctx.train(sctx)
2020-07-18T03:21:53.6190209Z   File "/home/runner/work/dffml/dffml/model/daal4py/dffml_model_daal4py/daal4pylr.py", line 134, in train
2020-07-18T03:21:53.6190498Z     self.lm.compute(xdata, ydata)
2020-07-18T03:21:53.6190837Z   File "build/daal4py_cy.pyx", line 16947, in _daal4py.linear_regression_training.compute
2020-07-18T03:21:53.6191353Z   File "build/daal4py_cy.pyx", line 16929, in _daal4py.linear_regression_training._compute
2020-07-18T03:21:53.6191649Z RuntimeError: Number of rows in numeric table is incorrect
```

Reply:
@pdxjohnny  thanks. That was the issue.